README.md

```diff
@@ -10,19 +10,18 @@
   <a aria-label="npm package" href="https://www.npmjs.com/package/@primer/components">
     <img alt="" src="https://img.shields.io/npm/v/@primer/components.svg">
   </a>
-  <a aria-label="contributors graph" href="https://github.com/primer/components/graphs/contributors">
-    <img src="https://img.shields.io/github/contributors/primer/components.svg">
+  <a aria-label="contributors graph" href="https://github.com/primer/react/graphs/contributors">
+    <img src="https://img.shields.io/github/contributors/primer/react.svg">
   </a>
-  <a aria-label="last commit" href="https://github.com/primer/components/commits/main">
+  <a aria-label="last commit" href="https://github.com/primer/react/commits/main">
     <img alt="" src=
-  "https://img.shields.io/github/last-commit/primer/components.svg">
+  "https://img.shields.io/github/last-commit/primer/react.svg">
   </a>
-  <a aria-label="license" href="https://github.com/primer/components/blob/main/LICENSE">
-    <img src="https://img.shields.io/github/license/primer/components.svg" alt="">
+  <a aria-label="license" href="https://github.com/primer/react/blob/main/LICENSE">
+    <img src="https://img.shields.io/github/license/primer/react.svg" alt="">
   </a>
 </p>
 
-
 ## Documentation
 
 Our documentation site lives at [primer.style/components](https://primer.style/components). You'll be able to find detailed documentation on getting started, all of the components, our theme, our principles, and more.
```

Comment:
Should we investigate renaming this (down the road) as well?

Reply:
Semi-related: Do we anticipate renaming the npm package from `@primer/components` → `@primer/react`?

Reply:
I think "yes" to both. 

Changing the website URL and package name will have bigger consequences so they'll need to be handled with more caution.